maintenance_request_sequence/views/maintenance_team.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="maintenance_team_view_form">
```

Comment:
Is it possible to add `code_prefix` field to team `tree` column? For now, the only way to edit this value for existing during team creation or modification is first switching to kanban view.

Reply:
Added